src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -458,6 +520,16 @@ public function get_item_schema() {
 						),
 					),
 				),
+				'theme_uri'      => array(
+					'description' => __( 'The URI of the theme\'s webpage.' ),
+					'type'        => 'string',
+					'readonly'    => true,
```

Comment:
And `format: 'uri'` here as well.

Reply:
9282f21fe74989b06b5b3d359d281ea595c082e7